File: corpus/2018q2/_p2q2_ok/_check/F;0468.py
int A[]
int fileiras, cadeiras
int fs, cs
print("Quantas fileiras estão vendidas? ")
fileiras = int(input())
fs = 20 - fileiras
print("Quantas cadeiras estão vendidas? ")
cadeiras = int(input())
cs = 220 - cadeiras
print(" Estão ocupadas "+ fileiras +" Fileiras, e estão ocupadas "+ cadeiras +" Cadeiras")
print(" Estao sobrando "+fs + " Fileiras e "+cs +" cadeiras")

File: corpus/2018q2/_p2q2_ok/_check/F;0468.java

import java.util.Scanner;
public class Questao2 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        int A[];
        int fileiras, cadeiras;
        int fs, cs;
        System.out.println("Quantas fileiras estão vendidas? ");
        fileiras = entrada.nextInt();
        fs = 20 - fileiras;
        System.out.println("Quantas cadeiras estão vendidas? ");
        cadeiras = entrada.nextInt();
        cs = 220 - cadeiras;

        System.out.println(" Estão ocupadas "+ fileiras +" Fileiras, e estão ocupadas "+ cadeiras+" Cadeiras");
        System.out.println(" Estao sobrando "+fs+ " Fileiras e "+cs+" cadeiras");

    }

}
